runtime/doc/windows.txt

```diff
@@ -241,9 +241,9 @@ and 'winminwidth' are relevant.
 
 						*:hor* *:horizontal*
 :hor[izontal] {cmd}
-		Execute {cmd}.  Currently only makes a difference for
-		`horizontal wincmd =`, which will equalize windows only
-		horizontally.
+		Execute {cmd} in a horizontal split.  Has no effect on
```

Comment:
I don't think this is correct. It doesn't apply to `:edit` etc..

Reply:
I could say "only has an effect for `horizontal wincmd =` and |:terminal|". Felt weird to explicitly list out commands like that, but maybe that is best for now since that is indeed the case.

Reply:
Updated the text. Please re-review.